be/src/service/service_cn/starrocks_cn.cpp

```diff
@@ -0,0 +1,69 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
This file is nearly the same as `starrocks_be.cpp`, is it better to no duplicate it?